### DISCUSSION TITLE
How to convert mod_auth_openidc.dll to .so for windows

### DISCUSSION THREAD

**Discussion by bommavijju** [OpenIDC/mod_auth_openidc#1124] [Q&A]
We have Apache webserver for windows where the modules are in.so format.

When we are trying to enable OIDCCookieSameSite ON

We are getting error:

The Apache service named  reported the following error:
>>> Invalid command 'OIDCCookieSameSite', perhaps misspelled or defined by a module not included in the server configuration  

We have comiled the modauth open idc using below link which created dll file. We need .so file which we can add to modules folder.
Can you please guide us on created .so from .dll file

**Comment by zandbelt** [comment]
Windows support is only available under a commercial license
